frontend/src/components/pages/teacher/DistributeAssessmentPage.tsx

```diff
@@ -34,6 +34,8 @@ const DistributeAssessmentPage = (): React.ReactElement => {
   const [endDate, setEndDate] = useState<Date | null>(null);
   const [notes, setNotes] = useState("");
 
+  const [validDates, setValidDates] = useState(true);
```

Comment:
it looks like we're not setting this value. should we default false and then update value in the useEffect? or we can use `invalidStartDate` and `invalidEndDate` directly

Reply:
updated to set the value in the hook, but in this case i think we have to recompute some values in `useEffect`?